src/GitHubExtensionServer/Package.appxmanifest

```diff
@@ -255,6 +255,40 @@
                       </LightMode>
                     </ThemeResources>
                   </Definition>
+                  <Definition Id="GitHub_Releases" DisplayName="ms-resource:Widget_DisplayName_Releases" Description="ms-resource:Widget_Description_Releases" IsCustomizable="true">
+                    <Capabilities>
+                      <Capability>
+                        <Size Name="medium" />
+                      </Capability>
+                      <Capability>
+                        <Size Name="large" />
+                      </Capability>
+                    </Capabilities>
+                    <ThemeResources>
+                      <Icons>
+                        <Icon Path="Widgets\Assets\GitHubLogo_Light.jpg" />
+                      </Icons>
+                      <Screenshots>
+                        <Screenshot Path="Widgets\Assets\IssuesScreenshotLight.png" DisplayAltText="DevHomeGitHub icon" />

```

Comment:
We will need to generate some first screenshots of this before we release it. Adding this comment as a reminder.